package.json

```diff
@@ -4,7 +4,7 @@
     "@project-serum/anchor": "0.26.0",
     "@solana/spl-token": "0.3.7",
     "@solana/web3.js": "1.73.0",
-    "@uxd-protocol/uxd-client": "7.3.1-rc1"
+    "@uxd-protocol/uxd-client": "7.2.0-rc2"
```

Comment:
should it stay `7.3.1-rc1` as discussed?

Reply:
for sure, will revert 👌